packages/web/components/QuestChain/QuestHeading.tsx

```diff
@@ -108,14 +110,23 @@ const Heading: React.FC<Props> = ({
             value={questChain.quests.filter((q) => !q.paused).length.toString()}
           />
           <ChainStat
-            label="Date Created"
-            value={new Date(questChain.createdAt * 1000).toLocaleDateString(
-              'en-US',
+            label="Updated at"
+            value={moment(new Date(questChain.updatedAt * 1000)).format(
```

Comment:
We should probably start moving away from using moment https://momentjs.com/docs/#/-project-status/

Reply:
noted - do you suggest any alternatives? luxon?

Reply:
I believe I have used Luxon but not extensively so I can't really recommend it or against it